src/src.pro

```diff
@@ -61,7 +62,7 @@ msvc:QMAKE_CXXFLAGS += -MP
 # Mac specific flags.
 macx:QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.7
 
-QT += network opengl uitools multimedia gui
+QT += network opengl uitools multimedia gui gamepad
```

Comment:
Include this conditionally like before - I'll build all installers with Qt 5.9 but we have min 5.6 ATM for Linux distros.